person.rb

```diff
@@ -2,7 +2,7 @@
 
 # class Person
 class Person < Nameable
-  attr_reader :id, :parent_permission
+  attr_reader :id, :parent_permission, :rentals
```

Comment:
- [ ] Please add a method for adding rentals. It will ensure that there is a relationship between the Person using the intermediate class Rental.

See example:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
end
```

Kindly note that the `self` keyword is referencing the `Person` class.